common/djangoapps/student/forms.py

```diff
@@ -38,13 +38,18 @@ def send_account_recovery_email_for_user(user, request, email=None):
     """
     site = get_current_site()
     message_context = get_base_template_context(site)
+    logistration_mfe_enabled = configuration_helpers.get_value(
+        'ENABLE_LOGISTRATION_MICROFRONTEND', settings.FEATURES.get('ENABLE_LOGISTRATION_MICROFRONTEND')
+    )
+    site_name = settings.LOGISTRATION_MICROFRONTEND_URL.split('//')[1] if logistration_mfe_enabled \
+        else configuration_helpers.get_value('SITE_NAME', settings.SITE_NAME)
```

Comment:
`should_redirect_to_logistration_mircrofrontend()` util function can be used. Also maybe we can save the logistration MFE domain name in a setting instead of calling split 

```suggestion
    site_name = settings.LOGISTRATION_MICROFRONTEND_URL_DOMAIN if should_redirect_to_logistration_mircrofrontend() \
        else configuration_helpers.get_value('SITE_NAME', settings.SITE_NAME)
```

Reply:
Done